src/Provider/GoogleMaps/Model/SubLocalityLevel.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider\GoogleMaps\Model;
+
+/**
+ * Class SubLocalityLevel
```

Comment:
This class docs does not give any value. Try to motivate why we need this class and what it is used for.